wiki/Guides/Compressing_files/en.md

```diff
@@ -58,20 +68,22 @@ This section will show you how to remove audio from videos using [Handbrake](htt
 
 ### Using FFmpeg
 
-This section will show you how you can use [FFmpeg](https://ffmpeg.org/) to reduce video file size. FFmpeg is a program used through a [command-line interface (CLI)](https://en.wikipedia.org/wiki/Command-line_interface), meaning it does not have any [graphical user interface (GUI)](https://en.wikipedia.org/wiki/Graphical_user_interface) by itself. While this may seem intimidating, FFmpeg can offer more flexibility than other tools, such as being able to re-encode multiple video files more quickly and easily.
+**FFmpeg is a program used through a [command-line interface (CLI)](https://en.wikipedia.org/wiki/Command-line_interface), meaning it does not have any [graphical user interface (GUI)](https://en.wikipedia.org/wiki/Graphical_user_interface) by itself.** While this may seem intimidating, FFmpeg can offer more flexibility than other tools, such as when integrated into a script.
```

Comment:
I don't think it makes sense to add emphasis here (or the GUI wikipedia link, too)
```suggestion
FFmpeg is a program used through a [command-line interface (CLI)](https://en.wikipedia.org/wiki/Command-line_interface), meaning it does not have any graphical interface by itself. While this may seem intimidating, FFmpeg can offer more flexibility than other tools, such as when integrated into a script.
```